File: reel/cmd/sox.py
"""Use the sox play utility."""
from .._spool import Spool


def speakers():
    """Send audio to the speaker."""
    cmd = 'play'
    flags = ['-t', 'raw',
             '-r', '44.1k',
             '-e', 'signed-integer',
             '-b', '16',
             '--endian', 'little',
             '-c', '2',
             '-']
    return Spool(cmd, xflags=flags)

File: src/cmd/sox.rs
use std::process::Stdio;

use tokio::process::Command;

pub fn play() -> Command {
    let mut cmd = Command::new("play");
    cmd.args(&["-t", "raw"])
        .args(&["-r", "44.1k"])
        .args(&["-e", "signed-integer"])
        .args(&["-b", "16"])
        .args(&["--endian", "little"])
        .args(&["-c", "2"])
        .arg("-")
        .stdin(Stdio::piped())
        .stdout(Stdio::null())
        .stderr(Stdio::null());
    cmd
}
